behave_django/testcase.py

```diff
@@ -5,11 +5,23 @@
 
 
 class BehaviorDrivenTestCase(StaticLiveServerTestCase):
+
+    """Test case attached to the context during behave execution
+
+    This test case prevents the regular tests from running.
+    """
+
     def runTest(*args, **kwargs):
         pass
 
 
 class ExistingDatabaseTestCase(BehaviorDrivenTestCase):
+
+    """ Test case used for the --use-existing-database setup
```

Comment:
Sorry for being picky: There's a space before the beginning of the sentence.

And also there's a blank line between the docstring and the function signature (here and above). Is this on purpose?